admin/metabox/class-metabox.php

```diff
@@ -1154,6 +1155,23 @@ private function get_custom_fields_replace_vars( $post ) {
 		return $custom_replace_vars;
 	}
 
+	/**
+	 * Checks if we're past the date set for the Jetpack Boost ad.
+	 *
+	 * @return bool Whether the Jetpack Boost ad must be shown.
+	 */
+	private function checkJetpackBoostAdDate() {
+		$now                          = new DateTime( 'now' );
+		$jetpack_boost_ad_date_string = WPSEO_Options::get( 'jetpack_ad_start_date' );
+		$jetpack_boost_ad_date        = new DateTime( $jetpack_boost_ad_date_string );
```

Comment:
I think you should specify a timezone here too, or you might accidentally convert it